File: SMI-SDK/Java SDK/src/pt/sapo/gis/exception/EventCallException.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package pt.sapo.gis.exception;

/**
 *
 * @author GIS Team
 */
public class EventCallException extends TraceSDKException {

    public EventCallException() {
    }

    public EventCallException(String message) {
        super(message);
    }

    public EventCallException(Throwable cause) {
        super(cause);
    }

    public EventCallException(String message, Throwable cause) {
        super(message, cause);
    }
    
}

File: SMI-SDK/.NET SDK/Library/gis/exception/EventCallException.cs
using System;
namespace pt.sapo.gis.exception
{
    /**
     *
     * @author GIS Team
     */
    public class EventCallException : TraceException
    {

        public EventCallException() { }

        public EventCallException(String message) : base(message) { }

        public EventCallException(String message, Exception cause) : base(message, cause) { }

    }
}
